src/data/fintype.lean

```diff
@@ -190,6 +190,14 @@ def of_subsingleton (a : α) [subsingleton α] : fintype α :=
 
 end fintype
 
+lemma finset.card_univ [fintype α] : (finset.univ : finset α).card = fintype.card α :=
+rfl
+
+lemma finset.card_univ_diff [fintype α] [decidable_eq α] (s : finset α) :
+  (finset.univ \ s).card = fintype.card α - s.card :=
+eq.symm $ nat.sub_eq_of_eq_add $
+  by rw [← card_disjoint_union disjoint_sdiff, union_sdiff_of_subset (subset_univ _)]; refl
```

Comment:
There should be a general theorem about `(s \ t).card` when `t` is a subset of `s`

Reply:
Good catch will add it.